### ISSUE TITLE
Can't have two ppx_expect test libs in the same directory

### ISSUE THREAD

**Issue by j0sh** [ocaml/dune#4073]
## Expected Behavior

With multiple libraries in a directory, each with their own ppx_expect tests, dune should be able to run them all.

## Actual Behavior

Dune fails with "multiple rules are generated"

```
$ dune runtest
Entering directory '/home/josh/test'
Error: Multiple rules generated for
_build/.aliases/default/multi-inline/runtest-b11519c7100cda2ee17e767e4ea9f6b3:
- multi-inline/dune:12
- multi-inline/dune:5
```

## Reproduction

Test case below, as a gist that can be cloned: https://gist.github.com/j0sh/652ca9c24cc0a3cad18d7c4bce8529cd

(NB: While this can be worked around in practice by combining things into the same library or moving libraries into separate directories, this is not as easily done for projects where the structure predates Dune.)

Run `dune runtest` with the following:

```
; dune 
(library
  (name a)
  (modules ("a"))
  (preprocess (pps ppx_expect))
  (inline_tests)
)

(library
  (name b)
  (modules ("b"))
  (preprocess (pps ppx_expect))
  (inline_tests)
)
```

```ocaml
(* a.ml *) 
let foo () =
  print_endline "foo"

let%expect_test "foo" =
  foo ();
  [%expect {| foo |}]
```

```ocaml
(* b.ml *)
let bar () =
  print_endline "bar"

let%expect_test "bar" =
  bar ();
  [%expect {| bar |}]
```

## Specifications

- Version of `dune`

```
$ dune --version
2.6.2
```

- Version of `ocaml`

```
$ ocamlc --version
4.10.0
```

- Operating system (distribution and version):

CentOS 8.2

```
$ cat /etc/redhat-release 
CentOS Linux release 8.2.2004 (Core) 
```

**Comment by rgrinberg** [comment]
This was fixed in 2.7.0 I think.

**Comment by j0sh** [comment]
Confirmed that upgrading to 2.7.0 fixes the issue - thanks.
